CMakeLists.txt

```diff
@@ -2723,7 +2731,7 @@ if(WIN32 OR CYGWIN OR MSYS)
         endif(MSVC)
     endif(MSVC AND CMAKE_SIZEOF_VOID_P EQUAL 8)
 else(WIN32 OR CYGWIN OR MSYS)
-    install(TARGETS ${LIBRARIES_TO_INSTALL} DESTINATION ${CMAKE_INSTALL_FULL_LIBDIR})
```

Comment:
So what does this change do?  [The latest version of the documentation for the GNUInstallDirs module](https://cmake.org/cmake/help/v3.23/module/GNUInstallDirs.html) says that `CMAKE_INSTALL_<dir>` can be a relative path but `CMAKE_INSTALL_FULL_<dir>` is an absolute path, usually produced by prepending `[CMAKE_INSTALL_PREFIX](https://cmake.org/cmake/help/v3.23/variable/CMAKE_INSTALL_PREFIX.html#variable:CMAKE_INSTALL_PREFIX)`, but with some [special cases](https://cmake.org/cmake/help/v3.23/module/GNUInstallDirs.html#special-cases) handled.

Are the special cases relevant here?

Reply:
Using the absolute path will make the cmake_install.cmake generated install path with absolute path:
```
if("x${CMAKE_INSTALL_COMPONENT}x" STREQUAL "xUnspecifiedx" OR NOT CMAKE_INSTALL_COMPONENT)
  list(APPEND CMAKE_ABSOLUTE_DESTINATION_FILES
   "/usr/local/lib/libpcap.a")
  if(CMAKE_WARN_ON_ABSOLUTE_INSTALL_DESTINATION)
    message(WARNING "ABSOLUTE path INSTALL DESTINATION : ${CMAKE_ABSOLUTE_DESTINATION_FILES}")
  endif()
  if(CMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION)
    message(FATAL_ERROR "ABSOLUTE path INSTALL DESTINATION forbidden (by caller): ${CMAKE_ABSOLUTE_DESTINATION_FILES}")
  endif()
  file(INSTALL DESTINATION "/usr/local/lib" TYPE STATIC_LIBRARY FILES "/home/clement/work/PcapPlusPlus/Dist/_deps/pcap-build/libpcap.a")
endif()

```

Using relative path will generate a cmake_install.cmake like this where the PREFIX can still be defined at install time:
```
if("x${CMAKE_INSTALL_COMPONENT}x" STREQUAL "xUnspecifiedx" OR NOT CMAKE_INSTALL_COMPONENT)
  file(INSTALL DESTINATION "${CMAKE_INSTALL_PREFIX}/lib" TYPE STATIC_LIBRARY FILES "/home/clement/work/PcapPlusPlus/Dist/_deps/pcap-build/libpcap.a")
endif()
```
   
   
   
   Using the